src/main/java/org/broadinstitute/consent/http/resources/DarCollectionResource.java

```diff
@@ -80,21 +81,20 @@ public Response getCollectionsForUserByRole(@Auth AuthUser authUser, @PathParam(
     }
   }
 
-  @GET
-  @Path("role/{roleName}/summary")
-  @Produces("application/json")
-  @RolesAllowed({ADMIN, CHAIRPERSON, MEMBER, SIGNINGOFFICIAL, RESEARCHER})
-  public Response getCollectionSummariesForUserByRole(@Auth AuthUser authUser, @PathParam("roleName") String roleName) {
-    try {
-      User user = userService.findUserByEmail(authUser.getEmail());
-      validateUserHasRoleName(user, roleName);
-      // TODO: Replace with DarCollectionSummary when available
-      List<Object> summaries = darCollectionService.getSummariesForRoleName(user, roleName);
-      return Response.ok().entity(summaries).build();
-    } catch (Exception e) {
-      return createExceptionResponse(e);
-    }
-  }
+  // @GET
+  // @Path("role/{roleName}/summary")
+  // @Produces("application/json")
+  // @RolesAllowed({ADMIN, CHAIRPERSON, MEMBER, SIGNINGOFFICIAL, RESEARCHER})
+  // public Response getCollectionSummariesForUserByRole(@Auth AuthUser authUser, @PathParam("roleName") String roleName) {
+  //   try {
+  //     User user = userService.findUserByEmail(authUser.getEmail());
+  //     validateUserHasRoleName(user, roleName);
+  //     List<DarCollectionSummary> summaries = darCollectionService.getSummariesForRoleName(user, roleName);
+  //     return Response.ok().entity(summaries).build();
+  //   } catch (Exception e) {
+  //     return createExceptionResponse(e);
+  //   }
+  // }
```

Comment:
Is this intentional? This breaks the swagger page since the endpoint is removed.

Reply:
Yeah, I didn't want to enable an endpoint with no associated tests in a service method PR. I re-enable it in https://github.com/DataBiosphere/consent/pull/1660 since it adds the tests in.